File: Python_parte1/lista_1/ex8.py
n1 = int(input("Insira o primeiro número: "))
n2 = int(input("Insira o segundo número: "))

soma = 0

if n1 > n2:
    temp = n2
    n2 = n1
    n1 = temp

    for i in range(n1, n2):
        soma += i
elif n1 == n2:
    soma = 0
else:
    for i in range(n1, n2):
        soma += i

print(f"Soma = {soma}")

File: C_parte2/lista_4/ex8.c
#include <stdio.h>


int main(void)
{
    int i, n1, n2, soma = 0, temp;

    printf("Insira o primeiro número: ");
    scanf("%d", &n1);

    printf("Insira o segundo número: ");
    scanf("%d", &n2);

    if (n1 > n2)
    {
        temp = n1;
        n1 = n2;
        n2 = temp;
    }

    for (i = n1; i < n2; i++)
    {
        soma += i;
    }

    printf("Soma: %d\n", soma);

    return 0;
}
